docs/manuals/source/TasksAndConcepts/CatalogMaintenance.rst

```diff
@@ -434,6 +433,39 @@ PostgreSQL Database
    single: Database; PostgreSQL
    single: PostgreSQL
 
+.. _FreeSpacePostgres:
+
+Free space needed with PostgreSQL Database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To ensure that all PostgreSQL maintenance operations like vacuuming and reindexing roll out smoothly we highly recommend to keep at least the size of your database available as free space.
+
+Bareos has some routines creating huge temp file for its operation, which also needs free space.
```

Comment:
Which routines?

Reply:
bvfs_update, big accurate file list construction, virtual full and so on.

Reply:
Normally resolved with new wording.